scripts/notebook-instance-monitor/on-start.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -ex
+
+# OVERVIEW
+# Change the conditions in the python script to monitpr and stop notebook instances with this lifecycle config script
```

Comment:
This is not clear. For example, which conditions should the users change? Which python script?

Typo in `monitpr`.

Reply:
removed/updated